products/ibm-i.md

```diff
@@ -0,0 +1,167 @@
+
+-   releaseCycle: "7.4"
+    releaseDate: 2019-06-21
+    eol: false
+    latest: "7.4"
+    latestReleaseDate: 2019-06-21
+    link: https://www.ibm.com/support/pages/customer-notices-and-information-ibm-i-74
+
+-   releaseCycle: "7.3"
+    releaseDate: 2016-04-15
+    eol: 2023-09-30
+    latest: "7.3"
+    latestReleaseDate: 2016-04-15
+    link: https://www.ibm.com/support/pages/customer-notices-and-information-ibm-i-73
+
+-   releaseCycle: "7.2"
+    releaseDate: 2014-05-02
+    eol: 2021-04-30
+    latest: "7.2"
+    latestReleaseDate: 2014-05-02
+    link: https://www.ibm.com/support/pages/customer-notices-and-information-ibm-i-72
+
+-   releaseCycle: "7.1"
+    releaseDate: 2010-04-23
+    eol: 2018-04-30
+    latest: "7.1"
+    latestReleaseDate: 2010-04-23
+    link: https://www.ibm.com/support/pages/customer-notices-and-information-ibm-i-71
+
+-   releaseCycle: "6.1"
+    releaseDate: 2008-04-23
+    eol: 2015-09-30
+    latest: "6.1"
+    latestReleaseDate: 2008-04-23
+    link: https://www.ibm.com/support/pages/customer-notices-and-information-ibm-i-61
+
+-   releaseCycle: "5.4"
+    releaseDate: 2006-02-14
+    eol: 2013-09-30
+    latest: "5.4"
+    latestReleaseDate: 2006-02-14
+    
+
+-   releaseCycle: "5.3"
+    releaseDate: 2004-06-11
+    eol: 2009-04-30
+    latest: "5.3"
+    latestReleaseDate: 2004-06-11
+    
+-   releaseCycle: "5.2"
+    releaseDate: 2002-08-30
+    eol: 2007-04-30
+    latest: "5.2"
+    latestReleaseDate: 2002-08-30
+    
+-   releaseCycle: "5.1"
+    releaseDate: 2001-05-25
+    eol: 2005-09-30
+    latest: "5.1"
+    latestReleaseDate: 2001-05-25
+
+-   releaseCycle: "4.5"
+    releaseDate: 2000-07-28
+    eol: 2002-07-31
+    latest: "4.5"
+    latestReleaseDate: 2000-07-28
+
+-   releaseCycle: "4.4"
+    releaseDate: 1999-05-21
+    eol: 2001-05-30
+    latest: "4.4"
+    latestReleaseDate: 1999-05-21
+
+-   releaseCycle: "4.3"
+    releaseDate: 1998-09-11
+    eol: 2001-01-31
+    latest: "4.3"
+    latestReleaseDate: 1998-09-11
+
+-   releaseCycle: "4.2"
+    releaseDate: 1998-02-27
+    eol: 2000-05-31
+    latest: "4.2"
+    latestReleaseDate: 1998-02-27
+
+-   releaseCycle: "4.1"
+    releaseDate: 1997-08-29
+    eol: 2000-05-31
+    latest: "4.1"
+    latestReleaseDate: 1997-08-29
+
+-   releaseCycle: "3.7"
+    releaseDate: 1996-11-08
+    eol: 1999-06-30
+    latest: "3.7"
+    latestReleaseDate: 1996-11-08
+
+-   releaseCycle: "3.6"
+    releaseDate: 1995-12-22
+    eol: 1998-10-31
+    latest: "3.6"
+    latestReleaseDate: 1995-12-22
+
+-   releaseCycle: "3.2"
+    releaseDate: 1996-06-21
+    eol: 2000-05-31
+    latest: "3.2"
+    latestReleaseDate: 1996-06-21
+
+-   releaseCycle: "3.1"
+    releaseDate: 1994-11-25
+    eol: 1998-10-31
+    latest: "3.1"
+    latestReleaseDate: 1994-11-25
+
+-   releaseCycle: "3.0"
+    releaseDate: 1994-06-03
+    eol: 1997-05-31
+    latest: "3.0.5"
+    latestReleaseDate: 1994-06-03
+
```

Comment:
On https://www.ibm.com/support/pages/release-life-cycle are also cycles for 2.3, 2.2, and 2.1:
```suggestion

-   releaseCycle: "2.3"
    releaseDate: 1993-12-17
    eol: 1996-05-31
    latest: "2.3"
    latestReleaseDate: 1993-12-17
    
-   releaseCycle: "2.2"
    releaseDate: 1992-09-18
    eol: 1995-03-31
    latest: "2.2"
    latestReleaseDate: 1992-09-18

-   releaseCycle: "2.1"
    releaseDate: 1991-05-24
    eol: 1994-06-30
    latest: "2.1.1"
    latestReleaseDate: 1992-03-06
    
```
WDYT?